src/SDKs/HDInsight/Management/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -7,11 +7,11 @@
 		<PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
 		<Description>Azure HDInsight Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
-		<Version>4.4.0-preview</Version>
+		<Version>5.0.0</Version>
 		<PackageTags>Microsoft Azure HDInsight Management;HDInsight;HDInsight Management</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
-        This is a public preview release of the Azure HDInsight SDK. Add support for get/update gatewaysettings.
```

Comment:
It would be helpful if you point out the features being released in this stable version in the `PackageReleaseNotes`

Reply:
Good suggestion! I have updated the `PackageReleaseNotes`